README.md

```diff
@@ -0,0 +1,50 @@
+## CUBRID Database Management System
+CUBRID is a DBMS being supported by an active community of open source developers 
+and provides better performance and features necessary for Web services. 
+
+This Software is released under Apache License 2.0 and BSD according to CUBRID components.
+For brevity, CUBRID Server Engine is under Apache License 2.0 and CUBRID APIs and Connectors are under BSD License.
+For details, please refer to the CUBRID License Page(http://www.cubrid.org/cubrid).
+
+Below You will see the brief list of sections to guide You to easily get started. 
+
+## MAJOR REFERENCES
+- CUBRID Official Site: http://www.cubrid.org ,  http://www.cubrid.com
+- CUBRID Development Site(Global): http://jira.cubrid.org
+- CUBRID Manuals: http://www.cubrid.org/manuals 
+- CUBRID JDBC Manuals (V11.0) : https://www.cubrid.org/manual/en/11.0/api/jdbc.html (Eng)
+  https://www.cubrid.org/manual/ko/11.0/api/jdbc.html (Kor)
+
+## DOWNLOADS and FILE REPOSITORIES
+- http://www.cubrid.org/downloads
+- http://ftp.cubrid.org
+
+## HOW TO BUILD/INSTALL CUBRID JDBC Driver
+### CUBRID JDBC Driver Source Build Guide (ANT Build Project)
+- Windows
+  ```
+  build.bat 
+  build.bat clean 
+  ```
+    
+- Linux
+
+  ```
+  ./build.sh 
+  ./build.sh clean
+  ```
+
+### CUBRID JDBC Driver Installation Guide for Windows/Linux Users
+- CUBRID JDBC driver (JAR File) implements an interface to enable access 

```

Comment:
- CUBRID JDBC driver (JAR File) implements an interface to enable access 
+ implements an interface to enable access